server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/json/Serializer.scala

```diff
@@ -34,7 +34,7 @@ import org.apache.james.mailbox.model.{MailboxACL, MailboxId}
 import play.api.libs.functional.syntax._
 import play.api.libs.json._
 
-import scala.collection.Seq
+import scala.collection.{Seq => JsonSeq}
```

Comment:
Before I get the question, this `Seq` is used by `JsError`, compared to the other `Seq` (from `scala.collection.immutable` package`) that we use in the rest of the class. This import was creating issues for the serialization, so I had to differentiate both

Reply:
what about naming it `MutableSeq`?

Reply:
why Mutable ? it's not the one from `scala.collection.mutable`